static/scenes/railroad/js/scene.js

```diff
@@ -2,9 +2,27 @@
 goog.provide('app.Scene');
 
 const gltfLoader = new THREE.GLTFLoader();
+// Initialized in preload.
+let loadedScene;
 
 class Scene {
-  constructor(loadedScene, camera, animations) {
+
+  static async preload() {
+    loadedScene = await gltfLoader.loadAsync('models/demo-scene-animated.glb');
```

Comment:
Nit: Worth passing in the model as a parameter? Eventually we'll want to be able to load new scenes, correct?

Reply:
The idea is that preload loads all the assets needed for the scenes, so when we add support for multiple scenes we should modify preload to load all the models here too.